### PR TITLE
add python implementation

### PR DESCRIPTION
**Pull request by jjliewie**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: content/4_Gold/Divisibility.mdx [hunk 1]
```diff
@@ -228,6 +229,22 @@ public class Divisors {
...
+from math import sqrt
+
+n = int(input())
+for i in range(n):
+	div_num = 0
+	x = int(input())
+	for j in range(1, int(sqrt(x) + 1)):
+		if x % j == 0:
+			div_num = div_num + 1 if j ** 2 == x else div_num + 2
+	print(div_num)
```
Comment (SansPapyrus683) [inline content/4_Gold/Divisibility.mdx:244]: can you make the var names consistent w/ the other solutions?

File: content/4_Gold/Divisibility.mdx [hunk 2]
```diff
@@ -228,6 +229,22 @@ public class Divisors {
 
 </JavaSection>
 
+<PySection>
+
+```py
+from math import sqrt
+
+n = int(input())
+for i in range(n):
```
Comment (SansPapyrus683) [inline content/4_Gold/Divisibility.mdx:238]:
```suggestion
for _ in range(int(input())):
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: content/4_Gold/Divisibility.mdx (hunk 1 above)
Comment (jjliewie) [reply content/4_Gold/Divisibility.mdx]: okay

**Review by SansPapyrus683 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]
⭐

**Pull request merged**